\

```diff
@@ -0,0 +1,167 @@
+package sortedset
```

Comment:
What's this file for? It seems to just be a duplicate of the old `sortedset.go` file.